templates_cspm/CloudAgentlessRole.yaml

```diff
@@ -0,0 +1,32 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: IAM Role for Agentless
+Parameters:
+  SysdigRoleName:
+    Type: String
+    Default: "SysdigAgentlessRole"
```

Comment:
Is this default consistent?

Reply:
renamed it to default created by terraform.

Reply:
renamed it.